code/game/objects/structures/rlattice.dm

```diff
@@ -0,0 +1,7 @@
+/obj/structure/lattice/rlattice
+	name = "reinforced lattice"
+	desc = "A support lattice. these ones are reinforced to widstand extreme enviroments." //cosmetic latice for all of your industrial themes in lava or what not
```

Comment:
Capitalize, change "these ones are" to "This one is"
"withstand" not "widstand"
"environments" not "enviroments"